sql/core/src/test/resources/sql-tests/inputs/pgSQL/date.sql

```diff
@@ -351,13 +351,13 @@ SELECT f1 - date '2000-01-01' AS `Days From 2K` FROM DATE_TBL;
 
 -- skip test constructors
 -- test constructors
--- select make_date(2013, 7, 15);
--- select make_date(-44, 3, 15);
+select make_date(2013, 7, 15);
+select make_date(-44, 3, 15);
 -- select make_time(8, 20, 0.0);
```

Comment:
If you want to keep this as a comment. You need to add JIRA issue above this line. The following is the PostgreSQL result.
```
select make_time(8, 20, 0.0);
 make_time 
-----------
 08:20:00
(1 row)
```

Reply:
Oops. Please ignore the above. It's `make_time`. I was confused.

Reply:
Spark doesn't support the `TIME` type for now. Are we going to reach parity to PostgreSQL and support it. If so, I could open JIRA for that.